hopsangeneratorgui/include/hopsangeneratorgui/hopsangeneratorgui.h

```diff
@@ -49,6 +49,16 @@ class HopsanGeneratorGUI : public QWidget
 
     bool checkComponentLibrary(const QString& libraryXMLPath);
 
+    bool addComponentToLibrary(const QString &libraryXmlPath, const QString &cafPath);
+
+    bool addComponentToLibrary(const QString &libraryXmlPath, const QString &typeName, const QString &displayName, const QString &cqsType,
```

Comment:
We should use a class with all of these as members instead. But right now we have one version in the generator and another in the GUI. So we can wait until they are unified.